src/plugin.ts

```diff
@@ -139,12 +139,11 @@ export default class Plugin {
   validateFunctions(instance: ConcurrencyFunction): boolean {
     return !!(
       instance.provisionedConcurrency &&
-      instance.provisionedConcurrency > 0 &&
       instance.concurrencyAutoscaling &&
       ((typeof instance.concurrencyAutoscaling === 'boolean' &&
         instance.concurrencyAutoscaling) ||
         (typeof instance.concurrencyAutoscaling === 'object' &&
-          instance.concurrencyAutoscaling.enabled === true))
+          instance.concurrencyAutoscaling.enabled !== false))
```

Comment:
@claydanford this seems okay, but as the value is boolean, it would seem to me !== false would be true even if concurrencyAutoscaling.enabled was set to null. In the current case, only when concurrencyAutoscaling.enabled is true is this valid. In the new case concurrencyAutoscaling.enabled would pass if it was set to true or not set at all. Is this okay?

Reply:
Under the `concurrencyAutoscaling` object, we require the `enabled: true` boolean. It should not work without that flag, perhaps some functions conditionally set the flag and it would still deploy.